daft/io/_lance.py

```diff
@@ -30,7 +30,7 @@ def _lancedb_table_factory_function(
 
 
 @PublicAPI
-def read_lance(url: str, io_config: Optional[IOConfig] = None) -> DataFrame:
+def read_lance(url: str, io_config: Optional[IOConfig] = None, **kwargs: Any) -> DataFrame:
```

Comment:
Rather than kwargs, could you please add "version" as its own parameter, thanks!

Reply:
ok，I have already made the modifications, thank you